### ISSUE TITLE
[Service Integrations] ECS Version sync with stack version

### ISSUE THREAD

**Issue by muthu-mps** [elastic/integrations#4792]
Creating separate issues for different integrations with multiple ecs versions. Going forward once we have all the integrations with the current stable version this change will be done in a single commit.

## Checklist

+ Identify the ecs version in build file and in the ingest pipeline for every integrations
+ Go-through the changes introduced in each release before updating the version.
+ Update the ecs version in build.yml and set the version in ingest pipeline.
+ Make changes to the ecs fields if necessary.
+ Run the pipeline tests to generate the new sample document.
+ Run the system tests.
+ Verify the document is updated with the migrated fields if any.

## Tracking Updates

- [x] [#4787](https://github.com/elastic/integrations/issues/4787)
- [x] [#4788](https://github.com/elastic/integrations/issues/4788)
- [x] [#4789](https://github.com/elastic/integrations/issues/4789)
- [x] [#4790](https://github.com/elastic/integrations/issues/4790)
- [x] [#4791](https://github.com/elastic/integrations/issues/4791)

## Update to 8.5.1
- The first step is to streamline all the integrations to 8.5.1. This is completed in this iteration.
- This involves going through the release highlights of each ECS versions and making necessary changes.

## Key Findings
- Ingest pipeline processor to set ECS version is missing in **Mysql** slowlog and error datastream. Created an [issue](https://github.com/elastic/integrations/issues/5067).
- Ingest pipeline processor to set ECS version is missing in **Mongodb** log datastream. Created an [issue](https://github.com/elastic/integrations/issues/5069).
- ECS fields added in fields.yml in **Mysql** and **Postgresql**. Filled an [issue](https://github.com/elastic/integrations/issues/5068).
- **Empty package-fields.yml** files in postgresql. This files can be cleaned up.
- Nice to have **sample-events** for **log** datastream in Redis, Mysql, Postgresql, Kafka etc...

**Comment by muthu-mps** [comment]
@ruflin - 

- So far we have captured integrations with different ECS versions and updating the versions. While updating captured couple of issues which are reported in the key findings. `Mysql` and `postgresql` has the ECS fields in `fields.yml` which we would like to remove and add it to ecs.yml if required. As we are going to remove the `ecs.yml ` in  [Dynamic ECS mappings](https://github.com/elastic/elastic-package/pull/1073) this should be done before we enable dynamic mappings. 
- The `package-fields.yml` in postgresql is found empty. Can we cleanup those files ?

**Comment by ruflin** [comment]
For the empty `package-fields.yml` file, I would check with the @elastic/ecosystem team to see if this can be removed or is required.

**Comment by mrodm** [comment]
Related to those `package-fields.yml` files, I've checked those files and just contain an empty group definition. For instance:
```
 $ cat ./data_stream/bgwriter/fields/package-fields.yml
- name: postgresql
  type: group
```

As it is an empty group, they could be deleted. There is no special meaning or requirement there. @ruflin @muthu-mps
